instruqt/avoiding-installation-pitfalls/01-working-with-preflight-checks/assignment.md

```diff
@@ -28,22 +28,22 @@ One of the areas teams worry about most when distributing software
 to customer clusters is whether those clusters will be properly
 configured for the application to run. To address this challenge,
 the Replicated Platform provides for the implementation and
-management for preflight checks that can describe your expectations
+management of preflight checks that can describe your expectations
 and validate them. Preflights are part of the
 [Troubleshoot](https://troubleshoot.sh) open source project.
 
- What is a Preflight Check?
+What is a Preflight Check?
 ===========================
 
 Preflight checks analyze your customer's cluster as a client without
 installing anything into it. This allows customers to run Preflight
-checks at any time before installing your software and keep the
+checks at any time before installing your software, and keep the
 cluster unchanged. They codify the pre-requisites you would catalog
 in your documentation and make them executable.
 
 You define your checks in a YAML file that follows the same format
 as a Kubernetes object. The simplest preflight object looks like this,
-and it's in the file `empty-preflights.yaml`:
+and it is in the file `empty-preflights.yaml`:
```

Comment:
Removed a lot of contractions to make the labs sound more professional (my two cents)